src/x3f_image.c

```diff
@@ -115,9 +144,17 @@
 {
   uint32_t keep[4], keep_cols, keep_rows;
 
-  if (!x3f_get_camf_matrix(x3f, name, 4, 0, 0, M_UINT, rect)) return 0;
+  if (!strncmp(name, "Fake", 4)) {
+    if (!x3f_get_fake_camf_matrix(x3f, name, image, rect))
+      return 0;
+  } else {
+    if (!x3f_get_camf_matrix(x3f, name, 4, 0, 0, M_UINT, rect))
+      return 0;
+  }
+
```

Comment:
To make it a bit less dirty, I would rather use a lookup table with names of the virtual rectangles with corresponding function pointers here.